rules/style.css

```diff
@@ -106,17 +117,19 @@ nav.TOC.main {
 @media (max-width: 915px) {
   nav.TOC {
     position: relative;
-    width: 100vw;
+    max-width: 100%;
+    width: unset;
   }
 
   nav.TOC.main {
     width: 100vw;
   }
 
-  figure > p > img {
-    height: 100%;
-    width: 100%;
+  figure>p>img {
     max-width: 100%;
+    height: 100%;
+    margin: 1rem auto;
+    scale: 1.15;
```

Comment:
Just so I can get a bit smarter on CSS - why the scale property?

Reply:
Some images appeared quite small so to counter that